### DISCUSSION TITLE
关于对 SQLite3 设置编译选项问题讨论

### DISCUSSION THREAD

**Discussion by kamenomi-dev** [xmake-io/xmake#5054] [General]
在xmake.lua：
`
add_defines(SQLITE_ENABLE_COLUMN_METADATA)

add_requires(sqlite3)
add_requires(sqlitecpp)
`

我设置了这个宏来使SQLite3支持Sqlite3_Column_ 的函数，但是我编译的时候却提示上述相关函数的符号找不到

（相关源代码稍后补全）

**Comment by kamenomi-dev** [comment]
See https://t.me/tbooxorg/14937

**Reply by waruqi** [reply]
要在包里加，而不是在用户工程里加 

https://github.com/xmake-io/xmake-repo/blob/dev/packages/s/sqlite3/xmake.lua

加个 add_configs 处理下，参考其他包配置
